nbconvert/exporters/templateexporter.py

```diff
@@ -58,6 +63,29 @@
         'json_dumps': json.dumps,
 }
 
+
+# copy of https://github.com/jupyter/jupyter_server/blob/b62458a7f5ad6b5246d2f142258dedaa409de5d9/jupyter_server/config_manager.py#L19
+def recursive_update(target, new):
```

Comment:
Why not `from jupyter_server.config_manager import recursive_update` instead? I think you taught me that one, @maartenbreddels!

It would add a dependency but, in this respect, would that be any different from the `from jupyter_core.paths import jupyter_path` addition line 14?

Reply:
Adding `jupyter_server` to nbconvert as a dependency would be substantial and unwanted as it would dramatically increase the width of overall dependencies needed for the project. If there's shared code we prefer it be added to `jupyter_core` over time to avoid pull adjacent dependency requirements.

Reply:
Ok, thanks for explaining @MSeal.